tests/device_perf_tests/stable_diffusion/test_perf_stable_diffusion.py

```diff
@@ -178,8 +178,8 @@ def test_stable_diffusion_perf(device, batch_size, num_inference_steps, expected
             return_dict=return_dict,
             config=config,
         )
-        profiler.end(f"model_run_for_inference_{i}")
         ttnn_output = ttnn_to_torch(ttnn_output)
+        profiler.end(f"model_run_for_inference_{i}")
```

Comment:
Hmm I don't think we want to measure time to convert ttnn to torch at the end of the model

Reply:
We do, otherwise, due to the way FD works, you don't actually sync on the device and your e2e time doesn't measure the whole mode.